sections/deckbuilding.tex

```diff
@@ -145,6 +145,12 @@ \subsection*{\pagetarget{spells}{Spell Cards}}
 The Token cannot be used during the \textbf{same Round} when the Guild is constructed.\par
 Spells can be \pagelink{Trading}{traded} in Alliance and Cooperative Scenarios.
 
+During Combat, \textbf{only one Spell Card} may be played by each player \textbf{per Combat Round}.
```

Comment:
```suggestion
The Token cannot be used during the \textbf{same Round} when the Guild is constructed. Spells can also be \pagelink{Trading}{traded} in Alliance and Cooperative Scenarios.

During Combat, \textbf{only one Spell Card} may be played by each player \textbf{per Combat Round}.
```
To avoid having two one-sentence paragraphs in a row.